Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -61,7 +61,7 @@ namespace AzToolsFramework
             m_prefabUndoCache.Destroy();
         }
 
-        PrefabOperationResult PrefabPublicHandler::CreatePrefab(const AZStd::vector<AZ::EntityId>& entityIds, AZ::IO::PathView absolutePath)
+        PrefabOperationResult PrefabPublicHandler::CreatePrefabInMemory(const AZStd::vector<AZ::EntityId>& entityIds, AZ::IO::PathView filePath)
```

Comment:
Since you removed the requirement that filePath should be absolute here, what happens when a user calls this public function with a relative path? In this function we do 'm_prefabLoaderInterface->GenerateRelativePath' at one point. To keep behavior consistent, it might be a good idea to add the IsAbsolutePath check to this function as well. And let the function calculate the relative path.

Reply:
Fixed.